### ISSUE TITLE
Qwen3.5 Image Lora post-training

### ISSUE THREAD

**Issue by julian991** [axolotl-ai-cloud/axolotl#3600]
### Please check that this issue hasn't been reported before.

- [x] I searched previous [Bug Reports](https://github.com/axolotl-ai-cloud/axolotl/labels/bug) didn't find any similar reports.

### Expected Behavior

Currently, I am testing the scripts for qwen3.5 finetuning for images. The model merged with Lora should be returning text content.

### Current behaviour

There is no content in the text reply, when I query in image+text or text inputs. I checked it with the base Qwen3.5 to see if its an error in my config but it returns the expected text for given images and text.

### Steps to reproduce

1. Run 10% of epoch of the provided qwen3.5 script https://github.com/axolotl-ai-cloud/axolotl/blob/main/examples/qwen3.5/9b-lora-vision.yaml
2. Test the model after merging the lora adapters on any text prompts or image+text prompts

### Config yaml

```yaml

```

### Which Operating Systems are you using?

- [x] Linux
- [ ] macOS
- [ ] Windows

### Python Version

3.12

### axolotl branch-commit

main/e079cf1

### Acknowledgements

- [x] My issue title is concise, descriptive, and in title casing.
- [x] I have searched the existing issues to make sure this bug has not been reported yet.
- [x] I am using the latest version of axolotl.
- [x] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by rehan243** [comment]
This looks like a classic case of the Lora adapters not being correctly integrated or activated post-training. In our experience with similar vision+language models, after merging Lora weights, you often need to ensure that the inference pipeline explicitly loads or applies the adapter weights. 

Check if the merging step follows the correct procedure—sometimes, the change isn't reflected unless you explicitly load the merged weights with `load_state_dict()` in your inference script. Also, verify if the inference code supports vision+language prompts with the merged Lora; some implementations require specific flags or configurations to activate the adapters during inference. 

If you haven't already, try running a small test with the base model + Lora weights loaded separately, then compare the outputs to see if the merging step is effective. Lastly, confirm that your prompt handling code distinguishes between text-only and image+text prompts, as mishandling here could cause no output even if the model is working fine.
